pkg/commands/policy/policy.go

```diff
@@ -163,6 +163,29 @@ func (c *PolicyCommand) Process(ctx context.Context) error {
 		)
 	}
 
-	// TODO: assign principals as reviewers to current pull request
+	// Make a request per user and team to avoid uncaught behavior from the GitHub
+	// API, which does not assign any of the provided reviewers if any of the
+	// principals exist on the pending review list.
+	for _, u := range users {
```

Comment:
Given my recent struggles with decoupling github from our implementation, it might be good to find a way to abstract this now rather than later. IE, PolicyEngine interface or somethign that has a RequestReviewer function (if i were to run this locally it would be a no-op, if i say use the github policy engine it adds a reviewer to the PR, etc). We can talk more offline.